packages/core/src/link.js

```diff
@@ -5,37 +5,37 @@ import * as API from '@ucanto/interface'
  * @template {number} Code
  * @template {number} Alg
  * @param {Code} code
- * @param {API.MultihashDigest<Alg>} digest
- * @return {API.Link<unknown, Code, Alg, 1> & CID}
+ * @param {API.UCAN.MultihashDigest<Alg>} digest
+ * @return {API.Link<any, Alg>}
  */
 export const create = (code, digest) =>
   /** @type {any} */ (CID.createV1(code, digest))
 
 /**
  * @template {number} Alg
- * @param {API.MultihashDigest<Alg>} digest
- * @return {API.Link<unknown, 0x70, Alg, 0> & CID}
+ * @param {API.UCAN.MultihashDigest<Alg>} digest
+ * @return {API.Link<any, Alg>}
  */
 export const createV0 = (digest) => /** @type {any} */ (CID.createV0(digest))
 
 /**
  * Type predicate returns true if value is the link.
  *
- * @template {API.Link<unknown, number, number, 0|1>} L
+ * @template {API.Link<any, number>} L
  * @param {unknown} value
- * @returns {value is (L & CID)}
+ * @returns {value is (L)}
  */
 export const isLink = (value) =>
   value != null && /** @type {{asCID: unknown}} */ (value).asCID === value
 
 export const asLink =
-  /** @type {<L extends API.Link<unknown, number, number, 0|1>>(value:L|unknown) => (L & CID)|null} */
+  /** @type {<L extends API.Link<any, number>>(value:L|unknown) => (L)|null} */
```

Comment:
```suggestion
  /** @type {<L extends API.Link<unknown, number>>(value:L|unknown) => (L)|null} */
```

Reply:
Type 'unknown' does not satisfy the constraint 'Capabilities'.